lib/DDG/Spice/MissingKids.pm

```diff
@@ -0,0 +1,34 @@
+package DDG::Spice::MissingKids;
+
+use DDG::Spice;
+
+# meta data                                                                                                                                           
+primary_example_queries "missing kids";
+secondary_example_queries "missing children";
+description "Shows missing children from your state if you are located in the US.";
+name "Missing Kids";
+code_url "https://github.com/brianrisk";
+topics "social";
+category "location_aware";
+attribution github => ['https://github.com/brianrisk','Brian Risk'],
+            email => ['brian@geneffects.com','Brian Risk'],
+            web => ["https://www.geneffects.com", "Geneffects"],
+            twitter => "brianrisk";
+
+# defining our triggers                                                                                                                               
+triggers query_lc => qr/missing (?:kids?|child(ren)?)/;
```

Comment:
Let's use trigger words here. I know the regex is simpler, but the trigger words are more efficient.

You *could* use `missing` as a start trigger and use the regex in your `handle`, but I think the best way is to just list out all the phrases.